src/vp8_probs.h

```diff
@@ -336,4 +336,240 @@ static const int vp8_bits_per_mb[2][128] =
     }
 };
 
+static const unsigned short vp8_prob_cost[256] =
```

Comment:
Yes, as a general rule keep them in .c files.  It's a good practice to follow, even if the savings are small.  It's the principal.  On an embedded systems sometimes the savings can be great depending on the size of the data.